src/idl_gen_go.cpp

```diff
@@ -993,7 +993,7 @@ class GoGenerator : public BaseGenerator {
     int max = 0;
     for (auto it = enum_def.Vals().begin(); it != enum_def.Vals().end();
         ++it) {
-      const int length = (*it)->name.length();
+      const int length = static_cast<int>((*it)->name.length());
```

Comment:
These changes fix MSVC2019 error (#5235): `warning C4267:  'initializing': conversion from 'size_t' to 'int', possible loss of data`
I had correct this during rebase without an intermediate commit (sorry).

@aardappel 
Can we replace the MSVC2010 build by MSVC2017?
Do you have an upgrade plan?